python/tests/test_crawler_current_events.py

```diff
@@ -9,14 +9,14 @@
 @pytest.fixture
 def current_events_text():
     """This function creates an fixture with test edgar_current_events html data"""
-    with open("tests/fixtures/edgar_current_events.html", "rt") as file:
+    with open("tests/fixtures/EDGAR Current Events.html", "rt") as file:
```

Comment:
please don't have spaces in the file structure. It will make it harder to move around.

Reply:
Changed